src/Util/AdminObjectAclManipulator.php

```diff
@@ -51,32 +50,26 @@ class AdminObjectAclManipulator
     protected $maskBuilderClass;
 
     /**
-     * @param string $maskBuilderClass
-     *
      * @phpstan-param class-string $maskBuilderClass
      */
-    public function __construct(FormFactoryInterface $formFactory, $maskBuilderClass)
+    public function __construct(FormFactoryInterface $formFactory, string $maskBuilderClass)
     {
         $this->formFactory = $formFactory;
         $this->maskBuilderClass = $maskBuilderClass;
     }
 
     /**
      * Gets mask builder class name.
```

Comment:
Same, could you remove the comment useless of this class

When the method is `getFooBar`, a comment `Gets foo bar` has no interest.